RawVideo/RawVideo/CameraCaptureService.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  CameraCaptureService.swift
+//  RawVideo
+//
+//  Created by Yassir Bisteni Aldana on 01/02/24.
+//
+
+import Foundation
+import AVFoundation
+import CoreMedia
+import UIKit
+import SwiftUI
+import AzureCommunicationCalling
+
+class CameraCaptureService : CaptureService, AVCaptureVideoDataOutputSampleBufferDelegate
+{
+    var camera: AVCaptureDevice!
+    var captureSession: AVCaptureSession!
+    var previewLayer: AVCaptureVideoPreviewLayer!
+    
+    func Start(camera: AVCaptureDevice) -> Void
+    {
+        self.camera = camera
+        captureSession = AVCaptureSession()
+        captureSession.sessionPreset = AVCaptureSession.Preset.vga640x480
+
+        do
+        {
+            let videoInput = try AVCaptureDeviceInput(device: camera)
+            captureSession.addInput(videoInput)
+        }
+        catch let ex
+        {
+            print(ex.localizedDescription)
+            
+            return
+        }
+        
+        previewLayer = AVCaptureVideoPreviewLayer(session: captureSession)
+
+        let videoOutput = AVCaptureVideoDataOutput()
+        if (captureSession.canAddOutput(videoOutput))
+        {
+            captureSession.addOutput(videoOutput)
+        }
+        
+        let queue = DispatchQueue(__label: "CameraCaptureDelegate", attr: nil)
+        videoOutput.setSampleBufferDelegate(self, queue: queue)
+        videoOutput.videoSettings = ["kCVPixelBufferPixelFormatTypeKey" : UInt(kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange)]
```

Comment:
Nit: There is a type safe constant for this one 
```suggestion
        videoOutput.videoSettings = [kCVPixelBufferPixelFormatTypeKey: UInt(kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange)]
```

Reply:
i tried that before but you need to add String() cast or either make is a string hardcoded value